src/utils.jl

```diff
@@ -210,7 +210,7 @@ Determine if `id` is used and not bound (aka. free) in `expr`.
 """
 isfreein(id::String, expr::SBML.Math) = interpret_math(
     expr,
-    map_apply = any(rec.(x.args)),
+    map_apply = (x, rec) -> any(rec.(x.args)),
```

Comment:
This bug is me spoiled on haskells. I wish we had function types and a typechecker. :cry:

Reply:
> If I get it right, this can be tested just by evaluating the math in sbml test 00039

Yes.